app/views/declare-partnership/single/success.html

```diff
@@ -0,0 +1,106 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+    Declare a partnership with a school
+{% endblock %}
+
+{% block beforeContent %}
+    {{ govukBackLink({
+        text: "Back",
+        href: "/declare-partnership"
+    }) }}
+{% endblock %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div style="  border: 4px solid #00703c;padding: 20px;margin-top: 0; margin-bottom: 40px">
+                <p class="govuk-heading-m">School added</p>
+                <p style="color: #00703c"><strong>Your partnership with {{ data["declare-partnership-school-name"] }}
```

Comment:
Is there a class making something `govuk green`? If not can we make one and use it? 

https://design-system.service.gov.uk/styles/colour/